ecs-cli/modules/clients/aws/cloudformation/template.go

```diff
@@ -150,6 +150,11 @@ var template = `
       "Type" : "String",
       "Description" : "ECS Cluster Name",
       "Default" : "default"
+    },
+    "InstanceRole" : {
+      "Type" : "String",
+      "Description" : "Optional - Instance IAM Role. Required if --capability-iam flag not specified.",
```

Comment:
`Required if ...` part reads weird here. The template has no notion of the `--capability-iam` flag. Can we get rid of that sentence?